go/types/types.go

```diff
@@ -871,6 +871,14 @@ func isNamedType(typ Type) bool {
 	return typ.Underlying(false).Node != typ.Node
 }
 
+func isTypeAlias(obj *ast.Object) bool {
+	if obj.Kind != ast.Typ {
+		return false
+	}
+	ts, ok := obj.Decl.(*ast.TypeSpec)
+	return ok && ts.Assign != 0
```

Comment:
maybe should use IsValid() method:
```go
return ok && ts.Assign.IsValid()
```

Reply:
Thanks, I agree, will push an updated commit.